modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -141,7 +141,11 @@ export class Wallet implements IWallet {
     if (baseCoin?.supportsTss() && this._wallet.multisigType === 'tss') {
       switch (baseCoin.getMPCAlgorithm()) {
         case 'ecdsa':
-          this.tssUtils = new EcdsaUtils(bitgo, baseCoin, this);
+          if (walletData.multisigTypeVersion === 'MpcV2') {
```

Comment:
```suggestion
          if (walletData.multisigTypeVersion === 'MPCv2') {
```
Lets add it to the public-types too

Reply:
Let's do that as a follow-up?